cpp/marc_tools/bash_script_convert_core_json_data

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+# bash script used to unify arrays in the JSON data files we get from CORE
+# in order to use them as input for convert_json_to_mrc
+JSON_INPUT_FILE=$1
+JSON_OUTPUT_FILE=$2
+if [[ ! "$#" == 2 ]]; then
+    echo "Usage: $0 Input_File_Name Output_File_Name"
+    exit 1
+fi 
+if [ -e $JSON_OUTPUT_FILE ]; then
+  echo "File $JSON_OUTPUT_FILE already exists!"
+  read -p "Would you like to overwrite the File $JSON_OUTPUT_FILE ? [y/n] " -n 1 -r
```

Comment:
Usually we write our scripts to not require interactive usage.  You can either remove this test or test file descriptor 0 being interactive.